### ISSUE TITLE
Fix VAE training

### ISSUE THREAD

**Issue by turnmanh** [aai-institute/tfl-training-practical-anomaly-detection#54]
VAE training yields a suboptimal model. We're expecting better performance and think about the following:
- increase batch size while keeping the rel. high learning rate
- add Dense layer
